test/unit/wiki_extensions_child_pages_count_test.rb

```diff
@@ -4,19 +4,40 @@ class WikiExtensionsChildPagesCountTest < Redmine::HelperTest
   include ApplicationHelper
   include ERB::Util
 
-  fixtures :projects, :wikis, :wiki_pages, :wiki_contents, :wiki_content_versions, :issues
+  fixtures :projects, :wikis, :wiki_pages, :wiki_contents, :wiki_content_versions, :issues,
+           :roles, :enabled_modules, :users
+
+  def setup
+    super
+    @project = nil
+  end
 
   def test_child_pages_count_in_wiki_content
+    @project = Project.find(1)
     wiki_page = WikiPage.find(2)
     assert_equal '<p>3</p>', textilizable('{{child_pages_count}}', object: wiki_page.content)
   end
 
   def test_child_pages_count_in_wiki_content_version
+    @project = Project.find(1)
     wiki_page = WikiPage.find(2)
     assert_equal '<p>3</p>', textilizable('{{child_pages_count}}', object: wiki_page.content.versions.first)
   end
 
+  def test_child_pages_count_with_child_page
+    @project = Project.find(1)
+    wiki_page = WikiPage.find(2)
+    assert_equal '<p>1</p>', textilizable('{{child_pages_count(Child_1)}}', object: wiki_page.content)
```

Comment:
このときって`object`を渡す必要ってあります？

```suggestion
    assert_equal '<p>1</p>', textilizable('{{child_pages_count(Child_1)}}')
```

Reply:
fix: [de132cc](https://github.com/otegami/redmine_wiki_extensions/pull/5/commits/de132cca865a9c7352e0e072da375c93b27e0aa2) 渡す必要がなかったので修正しました。